src/dpe.py

```diff
@@ -192,8 +192,9 @@ def run(self, net):
     parser = argparse.ArgumentParser()
     parser.add_argument(
         "-n", "--net", help="The net name as it is in test/testasm.", default='large')
-    parser.add_argument(
-            "-t", "--tile", help ="The number of tiles as generated by compiler.", default=-1)
```

Comment:
I understand this feature doesn't work at the moment. Still it should not be removed - a previous pull request resolved this issue, see Lines 218-221 below.